examples/pipelines/virt-sysprep-updater/virt-sysprep-updater-pipeline.yaml

```diff
@@ -33,10 +33,10 @@ spec:
       params:
         - name: pvc
           value: "$(tasks.modify-data-object.results.name)"
-        - name: sysprepCommands
```

Comment:
Does this change user facing parameters?

Reply:
Yes it does.